sdks/csharp/queries.cs

```diff
@@ -1,29 +1,45 @@
-// ------------------ RUN QUERY (Basic) ---------------------
-// <RunQuery>
-    // Run a query for all twins   
-    string query = "SELECT * FROM DIGITALTWINS";
-    AsyncPageable<BasicDigitalTwin> result = client.QueryAsync<BasicDigitalTwin>(query);
-// </RunQuery>
+using Azure;
+using Azure.DigitalTwins.Core;
+using System;
+using System.Collections.Generic;
 
-// ------------------ RUN QUERY & LOOP THROUGH PAGEABLE RESULTS WITH TRY/CATCH (Full sample) ---------------------
-// <FullQuerySample>
-AsyncPageable<string> result = client.QueryAsync("Select * From DigitalTwins");
-try
+namespace DigitalTwins_Samples
 {
-    await foreach(BasicDigitalTwin twin in result)
+    class QueriesSample
```

Comment:
Which would you recommend?

Here's where the sample shows up in context in the docs: https://docs.microsoft.com/azure/digital-twins/how-to-query-graph#query-with-paging
(The class declaration actually isn't shown in the excerpt in documentation, but sharing in case it helps give an idea of which is more appropriate)